src/pinns_pde_solve.jl

```diff
@@ -654,6 +668,45 @@ function get_numeric_derivative()
         end
 end
 
+function get_numeric_integral(strategy, indvars, depvars, dict_indvars, chain, derivative)
+    integral =
+        (u, cord, phi, θ, integrating_variable, integrand, lb, ub, strategy=strategy, indvars=indvars, depvars=depvars)->
+            begin
+                integrand_ = NeuralPDE.build_symbolic_loss_function(nothing, indvars, depvars, phi, derivative, nothing, chain, θ, strategy, integrand = integrand)
```

Comment:
all actions with the parser must be performed before starting training. To call the `build_symbolic_loss`  in `integral` function means that it will be called every time during training, so it will take a very long time.